src/ReverseString.java

```diff
@@ -7,6 +7,9 @@ public static void main(String[] args){
         System.out.println(s);
         reverse(s);
 
+        reverse(-2);
+        System.out.println(getDigitCount(1));
+
     }
 
     public static void reverse(char[] s){
```

Comment:
Would recommend  changing the loop to
`for(i = 0; i < n / 2; i ++) {
    swap(s[i], s[n - i - 1])
}`